gulpfile.js

```diff
@@ -702,25 +702,27 @@ gulp.task('minified-pre', ['buildnumber', 'locale'], function () {
 });
 
 gulp.task('minified-post', ['minified-pre'], function () {
-  var viewerFiles = [
-    MINIFIED_DIR + BUILD_DIR + 'pdf.js',
-    MINIFIED_DIR + '/web/viewer.js'
-  ];
+  var pdfFile = fs.readFileSync(MINIFIED_DIR + '/build/pdf.js').toString();
+  var pdfWorkerFile =
+    fs.readFileSync(MINIFIED_DIR + '/build/pdf.worker.js').toString();
```

Comment:
I think we should use `BUILD_DIR` here and in the other parts of this patch to not hardcode that and to be consistent with other places in the Gulpfile.

Reply:
I was just moving existing code around, but I suppose it wouldn't hurt to change this :-)